com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -226,20 +231,45 @@ RenderTextureDescriptor GetStereoCompatibleDescriptor(int width, int height, Gra
 
         bool RequireSRGBConversionBlitToBackBuffer(CameraData cameraData)
         {
-            bool requiresSRGBConversion = Display.main.requiresSrgbBlitToBackbuffer;
-            // For stereo case, eye texture always want color data in sRGB space.
-            // If eye texture color format is linear, we do explicit sRGB convertion
-#if ENABLE_VR && ENABLE_VR_MODULE
-            if (cameraData.isStereoEnabled)
-                requiresSRGBConversion = !XRGraphics.eyeTextureDesc.sRGB;
-#endif
-            return requiresSRGBConversion;
+            return cameraData.requireSrgbConversion && m_EnableSRGBConversionIfNeeded;
+        }
+
+        // TODO: deprecate this version in favor of the above with load/store actions
```

Comment:
"in favor of the above"? What should be used instead?

Reply:
Fixed in 6881f1f